ui.apps/src/main/content/jcr_root/apps/asset-share-commons/clientlibs/clientlib-site/js/details/js/action-buttons.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright [2017]  Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*global jQuery: false, AssetShare: false */
+jQuery((function($,ns) {
+	"use strict";
+	$(document).ready(function() {
```

Comment:
If we just always run the checks on page load for all ASC pages, we can do away w/ this.
Worst case it looks for the add/remove asset share IDs and doesnt find any DOM elements...

Reply:
Removed this js file